README.md

```diff
@@ -26,28 +26,104 @@
 =================
 
 # DataSketches Java Memory Component
- The goal of this component of the library is to provide high performance access to heap memory or 
- native memory for primitives, primitive arrays, ByteBuffers and memory mapped files. 
- This package is general purpose, has minimal external runtime dependencies and can be used in any 
- application that needs to manage data structures outside the Java heap.
+The goal of this component of the DataSketches library is to provide high performance access to heap memory or native memory for primitives, primitive arrays, ByteBuffers and memory mapped files. 
+This package is general purpose, has minimal external runtime dependencies and can be used in any 
+application that needs to manage data structures outside the Java heap.
```

Comment:
I do not know of any runtime dependencies - we removed the dependency on `slf4j`, and the module declaration does not require any modules outside of those provided by Java.  Should this change to `no external runtime dependencies`?